src/app/components/client/PercentageChart.tsx

```diff
@@ -93,29 +95,37 @@ export const PercentageChart = (props: WelcomeToPremiumProps) => {
               },
             })}
           </text>
-          {l10n.getFragment("exposure-reduction-chart-explanation", {
-            elems: {
-              label_line_1: (
-                <text
-                  className={styles.headingLabel}
-                  fontSize={headingLabelSize}
-                  x={diameter / 2}
-                  y={diameter / 2 + headingLabelSize + headingGap / 2}
-                  textAnchor="middle"
-                />
-              ),
-              label_line_2: (
-                <text
-                  className={styles.headingLabel}
-                  fontSize={headingLabelSize}
-                  x={diameter / 2}
-                  y={diameter / 2 + (headingLabelSize + headingGap / 2) * 2}
-                  textAnchor="middle"
-                />
-              ),
+          {l10n.getFragment(
+            /* c8 ignore next 5 */
+            // As the `SetExpectationsForUsers` feature flag is removed, the
+            // branch will be covered again:
+            props.enabledFeatureFlags.includes("SetExpectationsForUsers")
+              ? "exposure-reduction-chart-explanation"
+              : "exposure-reduction-chart-explanation-deprecated",
+            {
+              elems: {
+                label_line_1: (
+                  <text
+                    className={styles.headingLabel}
+                    fontSize={headingLabelSize}
+                    x={diameter / 2}
+                    y={diameter / 2 + headingLabelSize + headingGap / 2}
```

Comment:
Nit: We could pull the calculation for `y` out into a variable and reuse it in `L121`.

Reply:
I'm a bit wary about that since I didn't touch that code in this PR (I only changed the indentation because of the additional ternary), and the other y is slightly different (it has `* 2`), so I don't know whether I can capture the intent there well.